.github/workflows/elixir.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        elixir: [1.10.4, 1.11.2, 1.12]
+        elixir: [1.12]
```

Comment:
no need to support the older versions I think, also I've started using the is_struct guard which was introduced in 1.12

Reply:
why not start using 1.13?